plugin/src/main/java/com/yelp/codegen/KotlinGenerator.kt

```diff
@@ -402,14 +402,20 @@ open class KotlinGenerator : SharedCodegen() {
     ): CodegenOperation {
         val codegenOperation = super.fromOperation(path, httpMethod, operation, definitions, swagger)
 
-        retrofitImport.get(codegenOperation.httpMethod)?.let { codegenOperation.imports.add(it) }
+        retrofitImport[codegenOperation.httpMethod]?.let { codegenOperation.imports.add(it) }
         codegenOperation.allParams.forEach { codegenParameter: CodegenParameter ->
             codegenParameter.collectionFormat?.let {
                 val importName = "$toolsPackage.${it.toUpperCase()}"
                 if (importName !in codegenOperation.imports) {
                     codegenOperation.imports.add(importName)
                 }
             }
+
+            if (codegenParameter.isFile) {
+                codegenOperation.imports.add("okhttp3.RequestBody")
+                // The generated Retrofit APIs use RequestBody and not File objects
+                codegenOperation.imports.remove("File")
+            }
```

Comment:
I understand that you're wrapping the `"okhttp3.RequestBody"` here as otherwise you'll have a bunch of unused imports. On the other hand you're adding a new endpoint to `junit-test` that is not covered by a test. I'd suggest you add the new endpoint in a separate PR. Ideally we should tackle also the Multipart scenario that I'm pretty sure we don't cover.

Reply:
I'd rather keep the already existing endpoint, I'll try to see how expensive would be creating a test for it.

Reply:
Creating a realistic test for it seems going a bit beyond the scope of the PR, especially considering that I'm getting the feeling that files are not properly handled (#99 )

Going to offload the effort in a follow-up PR